app/assets/stylesheets/content/_select2.scss

```diff
@@ -200,3 +195,7 @@ $se2-width: 100%;
     }
   }
 }
+
+input[type="text"].select2-input {
```

Comment:
I guess i can ignore it for the moment, but then again, my experience in CSS is sparse at best. Does this have any takedowns in the long run - and, more importantly, is there a fix for this if this was critical?